lib/backend/database/helper.dart

```diff
@@ -19,7 +18,6 @@ class EntityStatCacheHelper {
     return stat;
   }
 
-  Future<void> set(EntityStat entity) async {
-    isar.writeTxn(() => isar.entityStats.put(entity));
-  }
+  Future<void> set(EntityStat entity) async =>
```

Comment:
you can remove both async and await here, the method returns a future so should be fine just returning that

Reply:
the whole clipboard viewer directory?

Reply:
done